src/server/server.ts

```diff
@@ -47,5 +54,47 @@ export async function server(
     },
   });
 
+  server.get<{Params: {'*': string}}>('/reconciliation/*', (request, reply) => {
```

Comment:
Minor: The Getty uses `/reconcile` and the unofficial VIAF and GeoNames services too. I'm not sure what's best practice here (it's just a name, but we could go for the most common one).

Reply:
Changed URL to `/reconcile`.